kafka-client/src/main/java/dev/responsive/kafka/api/stores/ResponsiveStores.java

```diff
@@ -295,5 +299,47 @@ public static <K, V> Materialized<K, V, WindowStore<Bytes, byte[]>> windowMateri
     );
   }
 
-  private ResponsiveStores() { }
+  /**
+   * Create a {@link StoreBuilder} that can be used to build a Responsive
+   * {@link SessionStore} and connect it via the Processor API. If using the DSL, use
+   * {@link #sessionMaterialized} instead.
+   * <p>
+   *
+   * @param storeSupplier a session store supplier
+   * @param keySerde      the key serde. If null, the default.key.serde config will be used
+   * @param valueSerde    the value serde. If null, the default.value.serde config will be used
+   * @return a store builder that can be used to build a session store with the given options
+   * that uses Responsive's storage for its backend
+   */
+  public static <K, V> StoreBuilder<SessionStore<K, V>> sessionStoreBuilder(
+      final SessionBytesStoreSupplier storeSupplier,
+      final Serde<K> keySerde,
+      final Serde<V> valueSerde
+  ) {
+    return new ResponsiveStoreBuilder<>(
+        Stores.sessionStoreBuilder(storeSupplier, keySerde, valueSerde),
+        false
```

Comment:
This should be set to the value of `truncateChangelog` that's in the session params

I'm pretty sure this was an existing bug though, we should probably fix that in a separate PR. We might want to be careful to ensure that Metronome sets this correctly -- @agavra do they still want to truncate the changelog? I'm a little hesitant after it saved us hahaha. Should we wait to fix this bug?

Reply:
I'm definitely supportive of not fixing it in this PR. I do think we should consider removing changelog truncation altogether.